packages/common/src/scopeSupportFacets/javascript.ts

```diff
@@ -10,12 +10,14 @@ const { supported, notApplicable } = ScopeSupportFacetLevel;
 export const javascriptScopeSupport: LanguageScopeSupportFacetMap = {
   list: supported,
   map: supported,
-  statement: supported,
   ifStatement: supported,
   regularExpression: supported,
   switchStatementSubject: supported,
   fieldAccess: supported,
 
+  statement: supported,
+  "statement.iteration": supported,
```

Comment:
This will absolutely get split up, and probably sooner rather than later. Should we do that now? Eg `statement.iteration.file` and `statement.iteration.block`? Tho `block` tends to mean something very different in our code base 🤔

Reply:
I was thinking about that, but there is no way to do separate scopes fixtures for that, both files will have 2 scopes I'm afraid.

Block is just the spoken form. We do call it paragraph as an id. I'm fine with block in this context.

Reply:
> I was thinking about that, but there is no way to do separate scopes fixtures for that, both files will have 2 scopes I'm afraid.

Yeah I'm not sure of a way around that. But I still think making it super explicit what the iteration scopes should be is preferred

> Block is just the spoken form. We do call it paragraph as an id. I'm fine with block in this context.

Sounds good

Reply:
Ok will split it!

Reply:
Fixed!